README.md

```diff
@@ -28,18 +28,20 @@ LibRealSense supported version: v2.50.0 (see [realsense2_camera release notes](h
 ## Installation Instructions
 
    ### Step 1: Install the ROS2 distribution
- - #### Ubuntu 18.04 : 
-   - [ROS2 Dashing](https://docs.ros.org/en/dashing/Installation/Ubuntu-Install-Debians.html)
-   - [ROS2 Eloquent](https://docs.ros.org/en/eloquent/Installation/Linux-Install-Debians.html)
+ - #### Ubuntu 22.04:
+   - [ROS2 Humble](https://docs.ros.org/en/humble/Installation/Ubuntu-Install-Debians.html)
  - #### Ubuntu 20.04: 
    - [ROS2 Foxy](https://docs.ros.org/en/foxy/Installation/Ubuntu-Install-Debians.html)
    - [ROS2 Galactic](https://docs.ros.org/en/galactic/Installation/Ubuntu-Install-Debians.html)
    - [ROS2 Rolling](https://docs.ros.org/en/rolling/Installation/Ubuntu-Install-Debians.html)

```

Comment:
I suggest moving Rolling out of the Ubuntu distribution title, Rolling should be above as the ROS2 up to date code (development)

Reply:
I will remove it at all from here, because we are not supporting Rolling distros really.
What do you think ?